views/views.go

```diff
@@ -28,6 +29,7 @@ func GetRouter() *chi.Mux {
 	// A good base middleware stack
 	r.Use(middleware.RequestID)
 	r.Use(middleware.RealIP)
+	r.Use(orangemiddleware.IpFilter)
```

Comment:
We can log the bad IP requests too. So, the IP filter can go after the timeout

Reply:
moved